File: OpenMP_part2/hybrid_helloworld.f90
program helloworld 
implicit none
include 'mpif.h'  
   integer :: omp_get_thread_num , omp_get_num_threads 
   integer :: myrank,ranks
   integer :: ierr
   call MPI_INIT(ierr)
   call MPI_COMM_SIZE(MPI_COMM_WORLD,ranks,ierr)
   call MPI_COMM_RANK(MPI_COMM_WORLD,myrank,ierr)

   write(6,*) 'Hello world'

   call MPI_FINALIZE(ierr)

end program helloworld

File: OpenMP_part2/hybrid_helloworld.c
#include "mpi.h"
#include "omp.h"
#include <stdlib.h>
#include <stdio.h>

#define MASTER 0 //Master process 
  
int main (int argc, char * argv[])
{
	int myrank,ranks;
	
	MPI_Init(&argc,&argv);
	MPI_Comm_size(MPI_COMM_WORLD,&ranks);
	MPI_Comm_rank(MPI_COMM_WORLD,&myrank);

	printf("Hello world\n");

	MPI_Finalize();

	return 0;
}

